### PR TITLE
Fixing the problem

### LINKED ISSUE #295: [pre-]selected layer is an image layer with no properties
Hi all,

there is a [_regression bug_](https://en.wikipedia.org/wiki/Software_regression) in the user-interface / convenience with the current version compared to 0.7.3:

If a user starts `Tools > Visualization > Plot measurements (ncp)`, they see an empty pulldown for X/Y axis, because the pre-selected layer is an image layer, that does not contain any features / measurements:

image

In 0.7.3 the window looks like this:

image

The current version makes it more complicated to use. I propose to change the behaviour back to how it was in 0.7.3: The pre-selected layer should be the currently active layer in the napari window.

## How to reproduce:

* Open blobs.tif
* Segment it using `Tools > Segmentation / Labeling > Gauss-Otsu-Labeling` (default parameters)
* Create measurements using `Tools > Measurement tables > Regionprops (skimage, nsr)`
* Run the plotter `Tools > Visualization > Plot measurements (ncp)`

Let me know if you need any further information from my side.

Best,
Robert

### PR DESCRIPTION
**Pull request by Cryaaa**
This PR fixes #295. I do a quick check which layers have properties or features attached to them and we select the first one in the layer list.

### CODE REVIEW

**Comment by haesleinhuepf** [comment]
Thanks gor quick fix @Cryaaa ! Would it be possible to select the layer that is currently active in napari instead? That would go in line with the suggestion in #295 and is the same behaviour as in the napari-assistant for example.

**Comment by Cryaaa** [comment]
> Thanks for the quick fix @Cryaaa ! Would it be possible to select the layer that is currently active in napari instead? That would go in line with the suggestion in #295 and is the same behaviour as in the napari-assistant for example.

Sure thats also a nice option and might require less code and avoid those errors popping up

File: napari_clusters_plotter/_Qt_code.py [hunk 1]
```diff
@@ -67,6 +67,9 @@ def layer_container_and_selection():
     layer_select = create_widget(annotation=Layer, label="layer")
     layer_selection_container.layout().addWidget(layer_select.native)
 
+    if viewer is not None and viewer.layers.selection.active is not None:
+        layer_select.value = viewer.layers.selection.active
```
Comment (haesleinhuepf) [inline napari_clusters_plotter/_Qt_code.py:71]: I'm speculating this crashes when two layers are selected in napari?

Comment (Cryaaa) [reply napari_clusters_plotter/_Qt_code.py:71]: I tested this and at the moment when you select two layers the selection.active returns none. I’m guessing that might change some time in the future though but I’m not sure

**Pull request merged**